projects/packages/publicize/src/class-social-admin-page.php

```diff
@@ -122,6 +122,9 @@ public function enqueue_admin_scripts() {
 		wp_dequeue_script( 'jetpack-social' );
 		wp_dequeue_style( 'jetpack-social' );
 
+		// TODO: Work out whether this should be done somewhere else, or only on WPCOM.
+		do_action( 'enqueue_block_editor_assets' );
```

Comment:
This may work but doesn't seem right for a generic admin page as it may have some unknown consequences when plugins/libs expect that action to be for the block editor.

Reply:
Indeed! I've done a lot more digging and it's because of a dependency on WPCOM 171102-ghe-Automattic/wpcom